web/core/components/issues/workspace-draft/quick-action.tsx

```
@@ -0,0 +1,160 @@
+      key: "make-a-copy",
+      title: "Make a copy",
+      icon: Copy,
+      action: () => {
+        setCreateUpdateIssueModal(true);
+      },
+    },
+    {
+      key: "move-to-issues",
+      title: "Move to issues",
+      icon: SquareStackIcon,
+      action: () => handleMoveToIssues && handleMoveToIssues(),
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)